secret_hitler/app.py

```diff
@@ -841,10 +853,10 @@ async def start_nomination(game : Game):
     await client.get_channel(game.channel_id).send(embed=embed)
 
 
-def get_category(guild):
+def __get_category(guild):
```

Comment:
why __ here?

Reply:
These seem like private methods to me. Internal helpers wrapping access to Discord. `__` is how you indicate private scope in Python. I figured I'd mark these up as private methods as I go.